pcbnew2boardview.py

```diff
@@ -139,17 +139,40 @@ def convert(pcb, brd):
     brd.write("\n")
 
 
+def convert_obdata(pcb, obdata):
+    obdata.write("COMPONENTS_DATA_START\n### Component Category Value Comment\n### v = value, p = package, c = manufacturer code, r = rating, m = misc, s = status\n###\n")
+    for module in pcb.GetFootprints():
+        ref = module.GetReference()
+        package = module.GetFPIDAsString()
+        value = module.GetValue()
+        
+        colon_search = package.find(':')
+        if colon_search >= 0:
+            package = package[colon_search+1:]
+            
+        obdata.write(f"{ref} p {package}\n")
+        if value == "NO-FIT" or value == "DNP":
+            obdata.write(f"{ref} s -\n")
+        obdata.write(f"{ref} v {value}\n")
+    obdata.write("COMPONENTS_DATA_END\n### END")
+
+
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument(
         "kicad_pcb_file", metavar="KICAD-PCB-FILE", type=str,
         help="input in .kicad_pcb format")
     parser.add_argument(
-        "brd_file", metavar="BRD-FILE", type=argparse.FileType("wt"),
+        "brd_file", metavar="BRD-FILE", type=argparse.FileType("w"),
         help="output in .brd format")
-
+    parser.add_argument(
```

Comment:
You should make this argument optional, or you're breaking backwards compatibility.

Reply:
Actually, you have not resolved this issue.

Reply:
Oh, I see, you're using `-obdata_file` as an option. That's not the right option format--use `--obdata` instead.

Reply:
Sorry for the oversight! It should be good to go now.

Reply:
You did not use the argument name I requested you use.

Reply:
Yes, sorry I missed that. It should match now 👍